src/frontend/screens/JoinProject.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react";
+import { Alert, StyleSheet, View } from "react-native";
+import { FormattedMessage, defineMessages } from "react-intl";
+import QRCode from "react-native-qrcode-svg";
+import { getUniqueId } from "react-native-device-info";
+
+import useWifiStatus from "../hooks/useWifiStatus";
+import { MEDIUM_BLUE, WHITE } from "../lib/styles";
+import { BackIcon } from "../sharedComponents/icons";
+import Button from "../sharedComponents/Button";
+import HeaderTitle from "../sharedComponents/HeaderTitle";
+import IconButton from "../sharedComponents/IconButton";
+import Text from "../sharedComponents/Text";
+import WifiBar from "../sharedComponents/WifiBar";
+
+const m = defineMessages({
+  title: {
+    id: "screens.JoinProject.title",
+    defaultMessage: "Join a Project",
+  },
+  instructionsTitle: {
+    id: "screens.JoinProject.instructionsTitle",
+    defaultMessage: "Instructions",
+  },
+  instructionsDescription: {
+    id: "screens.JoinProject.instructionsDescription",
+    defaultMessage: "Show this QR code to your Project Admin",
+  },
+  sendJoinRequest: {
+    id: "screens.JoinProject.sendJoinRequest",
+    defaultMessage: "Send Join Request instead",
+  },
+});
+
+const JoinProject = () => {
+  const { ssid } = useWifiStatus();
+
+  const deviceName: string =
+    "Android " + getUniqueId().slice(0, 4).toUpperCase();
+
+  return (
+    <View style={styles.pageContainer}>
+      {/* TODO: Probably want to do something smarter here if no ssid is detected */}
+      {ssid && <WifiBar deviceName={deviceName} ssid={ssid} />}
+      <View style={styles.container}>
+        <View>
+          <View style={styles.qrCodeContainer}>
+            <QRCode value="https://digital-democracy.org" size={250} />
+          </View>
+          <View style={styles.instructionsContainer}>
+            <Text style={styles.instructionsTitle}>
+              <FormattedMessage {...m.instructionsTitle} />
+            </Text>
+            <Text style={styles.instructionsDescription}>
+              <FormattedMessage {...m.instructionsDescription} />
+            </Text>
+          </View>
+        </View>
+        <Button
+          variant="text"
+          onPress={() =>
+            Alert.alert("Work in progress", "Feature not implemented yet")
```

Comment:
just a placeholder for now, since a different ticket will focus on implementing this